TaskManagement/timeline/services.py

```diff
@@ -0,0 +1,35 @@
+from project.models import Project
+from accounts.models import User
+import logging
+
+
+def get_dashboard(self, Dashboard):
+    dashboard = {}
+    try:
+        dashboard = super(Dashboard, self).get_context_data()
+        dashboard["total_user"] = User.objects.count()
```

Comment:
`
desi = ['a','b','c']
for i in desi:
dashboard[i] = your query
`
this will make your code very convincing let me know if you didnt get it